### PR TITLE
feat: create b2c web api

### PR DESCRIPTION
**Pull request by RasmusGraabaek**
## Description
New web app to handle b2c validation  and aggregated measure requests from the BFF toward EDI.
At the moment this only have a simple function and a swagger UI

### CODE REVIEW

File: source/B2CWebApi/B2CWebApi.csproj [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    <PropertyGroup>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <OutputType>Exe</OutputType>
+        <AssemblyName>Energinet.DataHub.EDI.B2CWebApi</AssemblyName>
+        <RootNamespace>Energinet.DataHub.EDI.B2CWebApi</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.16.1" />
```
Comment (lasrinnil) [inline source/B2CWebApi/B2CWebApi.csproj:11]: remove this

File: source/B2CWebApi/Responses/ResponseMessage.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace B2CWebApi.Responses;
+
+public class ResponseMessage
```
Comment (lasrinnil) [inline source/B2CWebApi/Responses/ResponseMessage.cs:17]: imo remove this class

File: source/B2CWebApi/Controllers/RequestAggregatedMeasureData.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// limitations under the License.
+
+using B2CWebApi.Responses;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Energinet.DataHub.EDI.B2CWebApi.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class RequestAggregatedMeasureData : ControllerBase
```
Comment (lasrinnil) [inline source/B2CWebApi/Controllers/RequestAggregatedMeasureData.cs:22]:
```suggestion
public class HelloWorldController: ControllerBase
```

File: source/B2CWebApi/Controllers/RequestAggregatedMeasureData.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+{
+    [HttpPost]
+    public new ResponseMessage Request()
+    {
+        return new ResponseMessage("Hello World");
```
Comment (lasrinnil) [inline source/B2CWebApi/Controllers/RequestAggregatedMeasureData.cs:27]: return "Hello World"

**Review by lasrinnil — APPROVED** [review]
Let's try to keep it as simple as possible to begin with. Great work!

**Review by lasrinnil — APPROVED** [review]

File: source/B2CWebApi/Properties/launchSettings.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:32723",
+      "sslPort": 44362
+    }
+  },
+  "profiles": {
```
Comment (MadsDue) [inline source/B2CWebApi/Properties/launchSettings.json:11]: do we need all the profiles?

**Review by Renetnielsen — APPROVED** [review]

**Pull request merged**